### PR TITLE
Re-balances acid damage

### PR DESCRIPTION
**Pull request by Diegoflores31**
# About the pull request

Changes acid splash damage . it will now be reduced with BIO armor and it will escale depending on your xeno TIER . 
formula = 20 + (TIERx5)

Reduces M3 light armor Bio resistance to be MEDIUMLOW instead of MEDIUM HIGH

Increases M3 Heavy armor Bio resistance to be MEDIUMHIGH instead of MEDIUM

# Explain why it's good for the game
Acid blood has been changing numbers pretty frequently over the years i think the issue is that is often too high or too low so now it adjusts depending on your tier so higher castes cant be just killed with knifes winout come consequences.

Acid blood now applies armored damage . this is to encourage heavier armor to those that want to take less damage from this kind of encounters.

Light armor BIO resistance has been reduced . the reasoning being that is the fastest armor available it makes little to no sense that is the one that protects you the most of them ALL variations .

Heavy armor BIO resistance was increased for those that wish to have a tankier playstyle be less affected.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
balance: Acid Splash damage increased to 25
balance: Acid Splash damage can be reduced with BIO armor.
balance: Reduced Light armor BIO resistance by 10
balance: Increased Heavy armor BIO resistance by 5
balance: Increased VL vest BIO resistance by 5
/:cl:

### CODE REVIEW

**Comment by MusselMain** [comment]
does this apply to suicide runners

**Comment by Diegoflores31** [comment]
> does this apply to suicide runners

no this is exclusively for acid splash ( the damage you get for shooting xenos too near)

File: code/modules/clothing/suits/marine_armor.dm [hunk 1]
```diff
@@ -549,7 +549,7 @@ var/list/squad_colors_chat = list(rgb(230,125,125), rgb(255,230,80), rgb(255,150
 	armor_bullet = CLOTHING_ARMOR_HIGH
 	armor_energy = CLOTHING_ARMOR_LOW
 	armor_bomb = CLOTHING_ARMOR_LOW
-	armor_bio = CLOTHING_ARMOR_NONE
+	armor_bio = CLOTHING_ARMOR_LOW
```
Comment (morrowwolf) [inline code/modules/clothing/suits/marine_armor.dm:552]: Make a new very low (5) or revert

Comment (morrowwolf) [reply code/modules/clothing/suits/marine_armor.dm:552]: Also document change

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Otherwise, willing to TM and try after changes.

File: code/modules/clothing/suits/marine_armor.dm (hunk 1 above)
Comment (Diegoflores31) [reply code/modules/clothing/suits/marine_armor.dm:552]: Will do tomorrow morning when i reach home

**Comment by Diegoflores31** [comment]
suggestions applied :3

**Review by Drulikar — APPROVED** [review]
Code changes look fine.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
I've thought about it, I don't think the tiered acid damage is really great and this is overall a nerf to acid damage in its current state.

Let's remove the tiered change and up the base acid blood damage by 5 and give it a whirl.

**Pull request merged**